### PR TITLE
feat(FEC-13211): Lower bar buttons behavior on smaller screen sizes

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves FEC-13211

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)

https://github.com/kaltura/playkit-js-ui/assets/62672270/488e245c-b636-49b9-a1e2-547c94a75de6

- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/bottom-bar/_bottom-bar.scss [hunk 1]
```diff
@@ -19,28 +19,34 @@
...
 
-  .left-controls {
-    float: left;
-    text-align: left;
-    pointer-events: none;
+  .controls-container {
+    width: 100%;
+
+    .left-controls {
+      float: left;
```
Comment (SivanA-Kaltura) [inline src/components/bottom-bar/_bottom-bar.scss:26]: is there a way to do it with flex ? float can behave unexpectedly.
same for the other uses of float.

File: src/components/bottom-bar/bettom-bar-utils.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+//@flow
```
Comment (SivanA-Kaltura) [inline src/components/bottom-bar/bettom-bar-utils.js:1]: bettem-bar-utils => bottom-bar-utils

File: src/components/time-display/time-display.js [hunk 3]
```diff
@@ -48,10 +48,10 @@ class TimeDisplay extends Component {
    * @returns {React$Element} - component
    * @memberof TimeDisplay
    */
-  render(props: any): React$Element<any> {
+  render({currentTime, duration, format = 'current / total'}: any): React$Element<any> {
```
Comment (SivanA-Kaltura) [inline src/components/time-display/time-display.js:51]: can total be 0 ?

File: src/components/bottom-bar/bettom-bar-utils.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+// eslint-disable-next-line require-jsdoc
+export function calculateControlsSize(controls: string[], currentControlWidth: number, playerWidth: number, isPlaylist: boolean): number {
+  let totalWidth = 0;
+  let controlWidth = 0;
+  for (let control of controls) {
+    if (control in SPATIAL_CONTROLS) {
+      controlWidth = SPATIAL_CONTROLS[control];
+    } else if (control === PlaybackControls.displayName) {
+      if (playerWidth > PLAYER_BREAK_POINTS.SMALL) {
+        controlWidth = isPlaylist ? currentControlWidth * 3 : currentControlWidth;
```
Comment (SivanA-Kaltura) [inline src/components/bottom-bar/bettom-bar-utils.js:16]: playlist is the only case in which we need this ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/bottom-bar/_bottom-bar.scss (hunk 1 above)
Comment (JonathanTGold) [reply src/components/bottom-bar/_bottom-bar.scss:26]: i try this but it conflicts with other css and break the player in other places, anyway it was already works with float, i want to change only what has to be changed

File: src/components/time-display/time-display.js (hunk 3 above)
Comment (JonathanTGold) [reply src/components/time-display/time-display.js:51]: its just format, any way the logic is the same as before not touched it , just syntax difference

File: src/components/bottom-bar/bettom-bar-utils.js (hunk 4 above)
Comment (JonathanTGold) [reply src/components/bottom-bar/bettom-bar-utils.js:16]: Right , only playlist has 2 additional controls (next & prev)

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**